docs/deployment/gardenlet_api_access.md

```diff
@@ -31,6 +31,7 @@ Today, the following rules are implemented:
 | Resource        | Verbs                                                         | Path                                          | Description |
 | --------------- | ------------------------------------------------------------- | ----------------------------------------------| ----------- |
 | `BackupBucket`  | `get`, `list`, `watch`, `create`, `update`, `patch`, `delete` | `BackupBucket` -> `Seed`                      | Allow `get`, `list`, `watch` requests for all `BackupBucket`s. Allow only `create`, `update`, `patch`, `delete` requests for `BackupBucket`s assigned to the `gardenlet`'s `Seed`. |
+| `BackupEntry`   | `get`, `list`, `watch`, `create`, `update`, `patch`           | `BackupEntry` -> `Seed`                       | Allow `get`, `list`, `watch` requests for all `BackupEntry`s. Allow only `create`, `update`, `patch` requests for `BackupEntry`s assigned to the `gardenlet`'s `Seed` and referencing `BackupBucket`s assigned to the `gardenlet`'s `Seed`. |
```

Comment:
No problem, if you plan to change the handling in the future then just keep in mind to also adapt this seed authorization/restriction coding. For now, I want to reflect the current state.

> Also, why is delete not allowed at all? Perhaps it's because gardenlet is not expected to delete BackupEntries

No strong reason, today, the gardenlet does not explicitly delete `BackupEntry` resources but rather relies on garbage collection after shoot deletion via owner references. Again, if this changes in the future, feel free to adapt if needed.